Added docs for new fingerprinting rule substitutions (#2864)
getsentry/sentry-docs: src/platforms/common/data-management/event-grouping/fingerprint-rules.mdx

message:"connection error*" -> connection-error, {{ tags.server_name }}
```

### `{{ message }}`

> This fills in the message of the event (similar to `error.value` but for
> captured messages). Note that this can produce really bad groups when
> messages are frequently changing:

Reviewer: Minor wording suggestion:
```suggestion
This fills in the message of the event (similar to `error.value` but for
captured messages). Note that this can produce groups with poor data quality if
messages are changing frequently:
```
